qiime_studio/api/plugins.py

```diff
@@ -51,7 +51,10 @@ def _build_data_dict(data):
             for name, type_ in value.signature.inputs.items()
         ]
         dict_[key]['parameters'] = [
-            {'name': name, 'type': repr(type_[0]), 'ast': type_[0].to_ast()}
+            {'name': name,
+             'type': repr(type_[0]),
+             'ast': type_[0].to_ast(),
+             'default': value.signature.defaults.get(name)}
```

Comment:
Ah, I didn't know `None` was able to be used to allow optionals now, that's awesome! I can rework this to account for that. Will need some frontend changes as well because right now it requires all fields to be supplied.